### ISSUE TITLE
Build fails with `Use of undeclared identifier '_CUPS_DNSSD_GET_DESTS'`

### ISSUE THREAD

**Issue by paulmenzel** [apple/cups#5105]
Building the latest commit on the master branch [on a GNU/Linux distribution][1], the build fails with the error below.

```
$ git log --oneline -1
befbadcef Fix ipptool -P output (some was going to stdout...)
$ ./configure
$ make -j
[…]
dest.c:2101:20: error: use of undeclared identifier '_CUPS_DNSSD_GET_DESTS'
  cupsEnumDests(0, _CUPS_DNSSD_GET_DESTS, NULL, 0, 0, (cups_dest_cb_t)cups_get_cb, &data);
                   ^
1 error generated.
```

[1]: https://travis-ci.org/paulmenzel/cups/jobs/272011837

**Comment by michaelrsweet** [comment]
This happens because you don't have mDNSResponder or Avahi. Fixed...

[master b39dd420c] Fix builds without Bonjour/Avahi (Issue #5105)

**Comment by paulmenzel** [comment]
Thank you for the quick reaction. Let’s see if Travis is happy now.
